Gems/AWSClientAuth/Code/Include/Authentication/AuthenticationProviderBus.h

```diff
@@ -29,7 +29,10 @@ namespace AWSClientAuth
         //! @return bool True if valid access token available, else False
         virtual bool IsSignedIn(const ProviderNameEnum& providerName) = 0;
 
-        //! Get cached tokens from last last successful sign-in for the provider.
+        //! [Deprecated] Get cached tokens from last successful sign-in for the provider.
```

Comment:
nit: Add note as to why deprecated / what replaced this functionality.

Reply:
Just updated the API description to include the reason and suggested solution.